superset/reports/api.py

```diff
@@ -574,7 +574,7 @@ def slack_channels(self, **kwargs: Any) -> Response:
         """
         try:
             params = kwargs.get("rison", {})
-            search_string = params.get("search_string")
+            search_string = [params.get("search_string", "").strip()]
```

Comment:
Any benefit for doing the search string cleanup here (and other place where you pass it to `get_channels_with_search`) as opposed to letting the `get_channels_with_search` be responsible for it in one place?

Reply:
My main concern here is really because we have this utility that converts a recipient string to a list, and so if we are going to pass around this search_string as a list, we should probably rename it as well.

Reply:
I think that's a valid concern. I can probably move this back there but still rely on the `util` we already have.

Reply:
Ohh, I know why I moved this outside. It's because during the v1 -> v2 migration, I need to know the list length before calling `get_channels_with_search` to confirm if all channels were found. Doing it outside and then again in the method felt repetitive, but considering this is specific to the migration, and this code will be removed soon as only V2 is going to be supported going forward I think that's ok.